packages/clarity-visualize/src/interaction.ts

```diff
@@ -72,25 +111,32 @@ export function pointer(event: Interaction.PointerEvent, iframe: HTMLIFrameEleme
     let type = event.event;
     let doc = iframe.contentDocument;
     let p = doc.getElementById("clarity-pointer");
-    let pointerWidth = 20;
-    let pointerHeight = 24;
+    let pointerWidth = config.pointerWidth;
+    let pointerHeight = config.pointerHeight;
 
     if (p === null) {
         p = doc.createElement("DIV");
         p.id = "clarity-pointer";
         p.style.position = "absolute";
-        p.style.zIndex = "10000000";
+        p.style.zIndex = `${config.zIndex + 2}`;
         p.style.width = pointerWidth + "px";
         p.style.height = pointerHeight + "px";
         doc.body.appendChild(p);
+
+        // Add custom styles
+        let style = doc.createElement("STYLE");
+        style.innerText = "@keyframes pulsate-one { 0% { transform: scale(1, 1); opacity: 1; } 100% { transform: scale(3, 3); opacity: 0; } }";
+        style.innerText += "@keyframes pulsate-two { 0% { transform: scale(1, 1); opacity: 1; } 100% { transform: scale(5, 5); opacity: 0; } }";
+        p.appendChild(style);
     }
 
-    p.style.left = (data.x - 8) + "px";
-    p.style.top = (data.y - 8) + "px";
+    p.style.left = (data.x - 5) + "px";
+    p.style.top = (data.y - 1) + "px";
```

Comment:
should these be config.pointerWidth and height instead of 5  and 1

Reply:
Yes, thanks. Done.